Sources/StreamChatTestTools/Models/ChatChannelMember_Mock.swift

```diff
@@ -9,22 +9,26 @@ public extension _ChatChannelMember {
     /// Creates a new `_ChatChannelMember` object from the provided data.
     static func mock(
         id: String,
-        isOnline: Bool,
-        isBanned: Bool,
-        userRole: UserRole,
-        userCreatedAt: Date,
-        userUpdatedAt: Date,
-        lastActiveAt: Date?,
-        extraData: ExtraData,
-        memberRole: MemberRole,
-        memberCreatedAt: Date,
-        memberUpdatedAt: Date,
-        isInvited: Bool,
-        inviteAcceptedAt: Date?,
-        inviteRejectedAt: Date?
+        name: String?,
```

Comment:
Why doesn't `name` have the default parameter?

Reply:
Forgot to add it 👍 Will fix it